homework/Stupns/homework10.py

```diff
@@ -0,0 +1,31 @@
+def tag(name):
+    def decorator(func):
+        def wrapper(text):
+            try:
+                if name == '123':
+                    return 'error'
+                else:
+                    return "<{0}>{1}</{0}>".format(name, func(text))
+            except TypeError:
+                err = 'error'
```

Comment:
no need to define one more variable, you can just print('error')

Reply:
okay